packetbeat/protos/drda/drda.go

```diff
@@ -0,0 +1,572 @@
+package drda
+
+import (
+	"fmt"
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
+	"strings"
+	"time"
+
+	"github.com/Intermernet/ebcdic"
+	"github.com/elastic/beats/packetbeat/procs"
+	"github.com/elastic/beats/packetbeat/protos"
+	"github.com/elastic/beats/packetbeat/protos/tcp"
+	"github.com/elastic/beats/packetbeat/publish"
+	"github.com/urso/ucfg"
```

Comment:
ucfg import can be removed after rebasing. Wrapper type *common.Config has been introduced in recent PRs